CardParts/src/Classes/Card Parts/CardPartAttributedTextView.swift

```diff
@@ -0,0 +1,291 @@
+//
+//  CardPartAttributedTextView.swift
+//  CardParts
+//
+//  Created by jloehr  on 12/9/19.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+public class CardPartUITextView: UITextView {
+    
+    /// prevents blue background from a selection
+    public override var selectedTextRange: UITextRange? {
+        get { return nil }
```

Comment:
Do we always want to return nil?

Reply:
Yes, we don't want selected text to display a background or indication of selection